app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1124,9 +1136,10 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         private const val AUTHENTICATION_DIALOG_TAG = "AUTH_DIALOG_TAG"
         private const val DAX_DIALOG_DIALOG_TAG = "DAX_DIALOG_TAG"
 
+        private const val MIN_PROGRESS = 10
         private const val MAX_PROGRESS = 100
-        private const val TRACKERS_INI_DELAY = 500L
-        private const val TRACKERS_SECONDARY_DELAY = 200L
+        private const val TRACKERS_INI_DELAY = 700L
```

Comment:
Something weird happened here if all the following code appear as changed in this PR.